job-dsls/src/main/resources/job-scripts/prod_create_images_templates_and_sources.jenkinsfile

```
@@ -0,0 +1,49 @@
+pipeline {
+    agent {
+        label 'kie-rhel7 && !master'
+    }
+
+    stages {
+        stage('Clean workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+
+        stage('Print variables') {
+            steps {
+                println "[INFO] VERSION: ${VERSION}"
+                println "[INFO] BUILDS: ${BUILDS}"
+                println "[INFO] OVERRIDING_FILE: ${OVERRIDING_FILE}"
+                println "[INFO] GITHUB_REFERENCE: ${GITHUB_REFERENCE}"
+            }
+        }
+
+        stage('Clone the repository') {
+            steps {
+                sh "git clone ${BXMS_TOOLING_REPOSITORY} bxms-tooling"
+            }
+        }
+
+        stage('Create images templates and sources') {
+            steps {
+                script {
+                    dir('bxms-tooling/release_scripts') {
+                        def OPTIONAL_PARAMS = ""
```

```suggestion
                        def OPTIONAL_PARAMS = GITHUB_REFERENCE != "" ? "--cref ${GITHUB_REFERENCE}" : ''
```

These lines can be removed, in case you accept this
```

                        if (GITHUB_REFERENCE != "") {
                            OPTIONAL_PARAMS = "--cref ${GITHUB_REFERENCE}"
                        }
```